kedro/framework/cli/pipeline.py

```diff
@@ -413,6 +421,9 @@ def package_pipeline(
     click.secho(message, fg="green")
 
 
+micropkg.add_command(package_pipeline)
```

Comment:
Also, related to what was raised earlier this week, I think we should show a deprecation warning that `kedro pipeline pull` is being deprecated in favour of `kedro micropkg pull` etc. So the two implementations would _slightly_ differ.